src/service/xhr-impl.js

```diff
@@ -485,7 +489,8 @@ export class FetchResponse {
                'responseType was not "arraybuffer"');
     dev.assert(!this.bodyUsed, 'Body already used');
     this.bodyUsed = true;
-    return Promise.resolve(this.xhr_.response);
+    return /** @type {!Promise<ArrayBuffer>} */ (
```

Comment:
although it currently is nullable https://github.com/ampproject/amphtml/blob/master/src/service/xhr-impl.js#L481 and we don't assert anywhere that it isn't. i believe xhr.response is nullable by default right? (on failures?)